changes.d/6835.feat.md

```diff
@@ -0,0 +1,3 @@
+Reformulate suicide triggers to expire tasks.
+Fixes a rare bug that could allow tasks to run after suicide triggering.
+The "expired" output is now completed on suicide triggered tasks.
```

Comment:
(this needs reformulating)

Reply:
That's still true as written. I'll add a note on custom completion conditions, if that's what you mean.

Reply:
I'm thinking about the experimental flag.

Reply:
Done.